core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -272,7 +272,7 @@ private[storage] class BlockManagerDecommissioner(
         migrationPeers.get(peer).foreach(_.running = false)
     }
     // If we don't have anyone to migrate to give up
-    if (migrationPeers.values.find(_.running == true).isEmpty) {
+    if (!migrationPeers.values.exists(_.running == true)) {
```

Comment:
`== true` is redundant too

Reply:
ok ~

Reply:
Let me check test dir today

Reply:
Address 16b2793 remove redundant  `== true` and  there are no other similar cases